package.json

```diff
@@ -102,6 +102,8 @@
     "sade": "^1.8.1",
     "semver": "^7.3.8",
     "sirv": "^2.0.2",
+    "vis-data": "^7.1.6",
+    "vis-network": "^9.1.6",
```

Comment:
I don't understand why you added theses dependencies?
For me theses deps belongs to the `vis-network` workspace only, may i missed something ?